src/pages/login.js

```diff
@@ -162,3 +167,5 @@ export default function Login() {
     </Grid>
   );
 }
+
+export default withRouter(Login);
```

Comment:
you're not using the router props in the component so you don't need with router HOC